### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by andrea-grapl**
Use URL fragments for routing using HashRouter from 'react-router-dom' rather than using LocalStorage. Tested successfully locally.

### CODE REVIEW

**Comment by andrea-grapl** [comment]
https://github.com/grapl-security/issue-tracker/issues/75

File: src/js/engagement_view/src/components/SideBarContent.tsx [hunk 1]
```diff
@@ -44,77 +48,127 @@ const useStyles = makeStyles({
...
-                    </TableRow>
+            <TableRow key={props.uid}>
+                <TableCell component="th" scope="row">
+                <Button className = {classes.lensName}
+                    onClick={
+                        () => { 
+                            props.setLens(props.lens)    
+                        }
+                }>
+                    {/* #TODO: change color of lens name based on score */}
```
Comment (colin-grapl) [inline src/js/engagement_view/src/components/SideBarContent.tsx:70]: Can we get an issue opened to track this?

File: src/js/engagement_view/src/modules/uploads/plugins/apiRequests.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ export const deletePlugin = async ( pluginName: string ): Promise <boolean> => {
             body: JSON.stringify({plugins_to_delete: [pluginName]})
         }
     );
-    const body = await res.json();
+    await res.json();
     return true;
```
Comment (colin-grapl) [inline src/js/engagement_view/src/modules/uploads/plugins/apiRequests.tsx:34]: Seems like we should be checking the response for success, not just returning true.

File: src/js/engagement_view/src/components/SideBarContent.tsx (hunk 1 above)
Comment (andrea-grapl) [reply src/js/engagement_view/src/components/SideBarContent.tsx:70]: Created issue: https://github.com/grapl-security/grapl/issues/132

**Pull request merged**